contributor/es/01-introduction-article-es.asciidoc

```diff
@@ -0,0 +1,63 @@
+Cuidar de la comunidad implica mantenerla saludable, elevarla a ella y a sus participantes, y defender sus necesidades en su organización.
+
+==== Product Owner
+
+El rol de https://innersourcecommons.org/learn/learning-path/product-owner[_Product Owner_] tiene cierta similitud con el rol de product owner de su proyecto.
+Sin embargo, existen diferencias: dependiendo del tamaño del proyecto, este rol a menudo lo desempeña la misma persona que actúa como responsable de confianza.
+En proyectos más grandes, o en equipos que solo usan parcialmente InnerSource para satisfacer sus necesidades al aceptar contribuciones, es probable que este rol lo desempeñe una persona separada de un trusted committer.
+
+Su interacción con el rol de product owner probablemente se centrará en determinar el ajuste de su contribución al producto general y su hoja de ruta.
+Puede trabajar con el rol de product owner para asegurarse de que los aspectos generales de la documentación, o la coherencia de UI / UX, se mantengan al combinar su contribución.
+
+Por último, pero no por ello menos importante, alguien que actúe como product owner podrá haber estado involucrado en llamar su atención sobre el proyecto, sus beneficios y la comunidad.
+
+Si desea conocer con más detalle de qué se tratan estos otros roles, y le animamos a que lo haga, hemos preparado secciones separadas sobre https://innersourcecommons.org/learn/learning-path/trusted-committer[_Trusted Committer_] y https://innersourcecommons.org/learn/learning-path/product-owner[_Product Owner_].
+
+=== Resumen de la sección
+
+En los siguientes 5 segmentos, aprenderá más en detalle sobre los diversos aspectos que se presentan aquí.
+
+El siguiente segmento detallará el _conjunto de ideas y hábitos_ que crean oportunidades para _convertirse en un Colaborador de InnerSource_.
+
+En el tercer segmento, analizaremos las _ética del colaborador_, es decir, los aspectos del _comportamiento_ que conducirán a un momento agradable y productivo para usted y el equipo anfitrión, y puede que generen más colaboraciones.
+La guest-in-home presentada en los videos introductorios servirá como ejemplo.
+
+El cuarto segmento describe las cosas prácticas que debe hacer para que su contribución sea un éxito: las _mecánicas de Contribución_.
+Le daremos consejos prácticos para aprovechar cuando se prepare para trabajar en una contribución, durante el desarrollo y también en la pull request.
+
+Una vez hayamos abordado los aspectos personales, los centrados en la interacción y los técnicos del rol de colaborador, el quinto segmento presenta los beneficios de hacer el esfuerzo de contribuir.
+Mostraremos los beneficios desde varias perspectivas: la suya, la de su equipo y la perspectiva de la empresa en general.
+
+El último segmento resumirá lo que hemos aprendido acerca de ser un colaborador de InnerSource.
```

Comment:
I'd remove the redundancy by moving the "online" bit from the options to the purpose: "...continuar el aprendizaje en línea de InnerSource, tanto con otros vídeos y artículos como mediante su participación en la comunidad de InnerSource" as although it deviates a bit from the original, it seems equivalent to the underlying message and more straight-forward.

Reply:
👍